coalib/misc/CheckVersion.py

```diff
@@ -0,0 +1,19 @@
+from pkg_resources import get_distribution, DistributionNotFound
```

Comment:
we should be removing dependency on `pkg_resources`, not adding more dependency on it.

and this type of functionality belongs in the dependency_management package.